File: task-03/Java/Sub task 2/subtask2.java
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;

public class subtask2 {
    public static void main(String[] args) {
        String inputFilename = "input.txt";
        String outputFilename = "output.txt";

        try {
            // Read the content of input.txt
            String content = new String(Files.readAllBytes(Paths.get(inputFilename)));

            // Write the content to output.txt
            Files.write(Paths.get(outputFilename), content.getBytes());

            System.out.println("Content successfully written to " + outputFilename);
        } catch (IOException e) {
            System.err.println("An error occurred: " + e.getMessage());
        }
    }
}

File: task-03/Rust/Sub task 2/subtask2.rs
use std::fs;
use std::io::{self, Write};

fn main() -> io::Result<()> {
    let input_filename = "input.txt";
    let output_filename = "output.txt";

    // Read the content of input.txt
    let content = fs::read_to_string(input_filename)?;
    
    // Write the content to output.txt
    let mut output_file = fs::File::create(output_filename)?;
    output_file.write_all(content.as_bytes())?;

    println!("Content successfully written to {}", output_filename);
    Ok(())
}

